core/tests/protractor/libraryFlow.js

```diff
@@ -27,6 +27,33 @@ var users = require('../protractor_utils/users.js');
 var waitFor = require('../protractor_utils/waitFor.js');
 var workflow = require('../protractor_utils/workflow.js');
 
+describe('Signin button test', function() {
+  beforeEach(function() {
+    browser.get('/splash');
+    waitFor.pageToFullyLoad();
+  });
+
+  it('should not show sign in button if user is logged in', function() {
+    users.createAndLoginUser('user123@signInButton.com', 'userSignIn');
```

Comment:
Hi @bansalnitish, I am not sure if `users.createAndLoginUser(..)` works on mobile (Browserstack) since I remember going for a workaround; feel free to go through the existing tests for mobile devices. Also, this PR needs to tested on Browserstack. @seanlip will be able to set you up with the credentials and you can find other instructions in `core/tests/.browserstack.env.example`. Ping me up for further help. Thanks!